neutron_policy_server/wsgi.py

```diff
@@ -80,12 +83,24 @@ def enforce_port_update():
         and ("mac_address" not in g.target["attributes_to_update"])
         and ("fixed_ips" not in g.target["attributes_to_update"])
     ):
+        LOG.info(
+            "No mac_address or fixed_ips in update targets for port "
+            f"{g.target['id']}, skip check."
+        )
         return Response("True", status=200, mimetype="text/plain")
 
     with db_api.CONTEXT_READER.using(g.ctx):
-        ports = port_obj.Port.get_objects(g.ctx, id=g.target["id"])
+        ports = port_obj.Port.get_objects(g.ctx, id=[g.target["id"]])
         if len(ports) == 0:
-            return Response("No match port found.", status=403, mimetype="text/plain")
+            # Note(ricolin): This happens with ports that are not well defined
+            # and missing context factors like project_id.
+            # Which port usually created by services and design for internal
+            # uses. We can skip this check and avoid blocking services.
+            LOG.info(
+                f"Can't fetch port {g.target['id']} with current "
+                "context, skip this check."
+            )
+            return Response("True", status=200, mimetype="text/plain")
```

Comment:
This is not blocking us, but I think we didn't consider context auth scope in previous version